frontend/client/styles/screens/add_member_modal.scss

```diff
@@ -56,6 +57,13 @@
     font-size: 16px;
     padding-left: 20px;
   }
+  .custom-select:after {
+    content: url('../../assets/arrow-down.svg');
+    position: relative;
+    right: 25px;
+    top: -1px;
+    background-color: transparent;
```

Comment:
The arrow itself is currently not clickable, which could be frustrating for the user.  Take a look at Isak's PR [here](https://github.com/covidwatchorg/portal/pull/381) for how this was solved on the Manage Team page.  I think it may just be the `pointer-events: none` property to the `:after`